pkg/account/account.go

```diff
@@ -39,6 +39,17 @@ func GetAccounts(key string, limit int, conn *sdk.Connection) ([]*v1.Account, er
 	return accounts.Items().Slice(), nil
 }
 
+func GetAccount(accountID string, conn *sdk.Connection) (*v1.Account, error) {
+	search := fmt.Sprintf("id = '%s'", accountID)
+
+	accounts, err := conn.AccountsMgmt().V1().Accounts().List().Search(search).Send()
```

Comment:
Can we use the AccountGet method instead? It should provide better performance than the search.